### PR TITLE
Feat/issue badge non can account

### PR DESCRIPTION
**Pull request by quocle108**
- allow 3rd-party(ram.can) pay ram for all action from users/issuers
- issue badge to non-CAN account by issued to ghost account, user can claim certification later once they has CAN account
- Removed offer/claim part which is no longer used

### CODE REVIEW

File: src/cryptobadge.cpp [hunk 1]
```diff
@@ -96,60 +110,46 @@ ACTION cryptobadge::issuebadge( name issuer, name owner, uint64_t badge_id, uint
...
-		check (offer_itr == _offert.end(), "offer already exist");
-
-		_offert.emplace( issuer, [&]( auto& s ) {     
-			s.cert_id = cert_id;
-			s.offered_to = owner;
-			s.owner = issuer;
-			s.date = current_time_point();
-		});
-	}
+	check (!(owner == ghost_account && require_claim == 1), "require_claim only owner == ghost account.");
```
[the remark this answers is not in the document]
Comment (danielAlvess) [reply src/cryptobadge.cpp:113]: I don't know why we need `require_claim` when we can use ghost_account to identify.

[the remark this answers is not in the document]
Comment (quocle108) [reply src/cryptobadge.cpp:113]: > I think it should be
> 
> ```c++
> check ((owner == ghost_account && require_claim == 1) || (owner != ghost_account && require_claim == 0), "require_claim only owner == ghost account.");
> ```

OK

[the remark this answers is not in the document]
Comment (quocle108) [reply src/cryptobadge.cpp:113]: > I don't know why we need `require_claim` when we can use ghost_account to identify.

To show to users that this badge can be claimed

[the remark this answers is not in the document]
Comment (quocle108) [reply src/cryptobadge.cpp:113]: BTW, the discussion for this feature still doesn't finish, so this is just a draft version

**Comment by quocle108** [comment]
@danielAlvess Pls verify again

**Pull request merged**